modules/mobile/app/controllers/mobile/v0/addresses_controller.rb

```diff
@@ -43,6 +43,7 @@ def validate
         # No domestic or military addresses should have a province but some have been coming in as a string 'null'
         suggested_addresses.each do |sa|
           if sa['address_type'].in?(['DOMESTIC', 'OVERSEAS MILITARY']) && sa['province'].present?
+            # i don't see this in logs. appears to be resolved. remove this
```

Comment:
Category: Debugging data
Action: remove. bug is fixed and this is no longer happening.